action.yml

```diff
@@ -10,49 +10,60 @@ inputs:
       The firmware stack to build.
       One of the following:
         - coreboot
+        - linux
+        - edk2
     required: true
-  path:
+  #-------------------------------
+  # Inputs common for all targets
+  sdk_version:
+    description: |
+      *sdk_version* specifies the docker toolchain tag to use
+      when building the image. This has an influence on the
+      IASL, GCC and host GCC version that is used to build
+      the target.
+      You must match the source level and sdk_version.
+      *Note:* Updating the sdk_version might result in different
+      binaries using the same source code.
+      For example:
+        coreboot_4.19:main
+        coreboot_4.19:latest
+        edk2-stable202111:latest
+      See https://github.com/orgs/9elements/packages
+    required: true
+  repo_path:
     description: |
-      *path* gives the (relative) path to the target (firmware) repository.
+      *repo_path* gives the (relative) path to the target (firmware) repository.
       If the current repository contains the selected target, specify: '.'
       Otherwise the path should point to the target (firmware) repository
       submodule that had previously been checked out.
     required: true
-  defconfig:
+  defconfig_path:
     description: |
-      *defconfig* gives the (relative) path to the defconfig that
+      *defconfig_path* gives the (relative to repo_path) path to the defconfig that
       should be used to build the target.
       For coreboot and linux this is a defconfig.
       For EDK2 this is a one-line file containing the build defines.
     required: true
-  platform:
+  output:
     description: |
-      *platform* specifies the DSC to use when building EDK2.
-      For example:
-        UefiPayloadPkg/UefiPayloadPkg.dsc
+      *output* specifies path (relative) to where place
```

Comment:
This sentence reads slightly odd. I'd change the word ordering to `*output* specifies the (relative) path where to place the produced files`

Reply:
nevermind